src/main/resources/api-specs/product-rating-openapi.yaml

```diff
@@ -95,14 +95,51 @@ components:
         - productId
         - rating
 
-    AverageProductRatingDto:
+    ProductReviewRatingStats:
       type: object
       properties:
         productId:
           type: string
           format: "uuid"
         rating:
           type: "double"
```

Comment:
Schema validation: Value should be one of: "array", "boolean", "integer", "null", "number", "object", "string"

Reply:
Yes, I saw this warning, but in the end it generates `Double` as expected, which LGTM.
I'll check if I can achieve the same result w/o violating the schema, though 👍

Reply:
Fixed